datahub-web-react/src/app/ingest/source/builder/CreateScheduleStep.tsx

```diff
@@ -141,29 +157,31 @@ export const CreateScheduleStep = ({ state, updateState, goTo, prev }: StepProps
                     )}
                 </Form.Item>
                 <StyledFormItem required label={<Typography.Text strong>Schedule</Typography.Text>}>
-                    <div style={{ paddingBottom: 10, paddingLeft: 10 }}>
-                        <AdvancedCheckBox type="secondary">Advanced</AdvancedCheckBox>
-                        <Checkbox
-                            checked={advancedCronCheck}
-                            onChange={(event) => setAdvancedCronCheck(event.target.checked)}
-                        />
-                    </div>
-                    {advancedCronCheck ? (
-                        <Input
-                            placeholder={DAILY_MIDNIGHT_CRON_INTERVAL}
-                            autoFocus
-                            value={scheduleCronInterval}
-                            onChange={(e) => setScheduleCronInterval(e.target.value)}
-                        />
-                    ) : (
-                        <Cron
-                            value={scheduleCronInterval}
-                            setValue={setScheduleCronInterval}
-                            clearButton={false}
-                            className="cron-builder"
-                            leadingZero
-                        />
-                    )}
+                    <Schedule>
+                        {advancedCronCheck ? (
+                            <CronInput
+                                placeholder={DAILY_MIDNIGHT_CRON_INTERVAL}
+                                autoFocus
+                                value={scheduleCronInterval}
+                                onChange={(e) => setScheduleCronInterval(e.target.value)}
+                            />
+                        ) : (
+                            <Cron
+                                value={scheduleCronInterval}
+                                setValue={setScheduleCronInterval}
+                                clearButton={false}
+                                className="cron-builder"
+                                leadingZero
+                            />
+                        )}
+                        <AdvancedSchedule>
+                            <AdvancedCheckBox type="secondary">Show Advanced</AdvancedCheckBox>
```

Comment:
Super nit but for some reason checkbox seems unintuitive.  I'd expect it on a new line with a chevron down (collapsible/accordion style)

Reply:
I felt that it was too much vertical with that and the "Runs at" stacked on one another.

Previously this checkbox was directly above the schedule builder, which I also didn't like..